website/docs/components/accordion/partials/code/how-to-use.md

```diff
@@ -0,0 +1,125 @@
+        Additional content for item one
+      </:content>
+    </A.Item>
+    <A.Item>
+      <:toggle>Item two</:toggle>
+      <:content>
+        Additional content for item two
+      </:content>
+    </A.Item>
+  </Hds::Accordion>
+```
+
+### Complex HTML content in toggle and content blocks
+```handlebars
+  <Hds::Accordion as |A|>
+    <A.Item @isOpen={{true}}>
+      <:toggle>
+        <Hds::Alert @type="compact" @color="success" as |A|>
+          <A.Title>Title</A.Title>
+          <A.Description>Plan finished <small>22 days ago</small></A.Description>
+        </Hds::Alert>
+      </:toggle>
+      <:content>
+        <p class="hds-typography-body-200">
+          <strong>Queued:</strong>
+          9 days ago >
+          <strong>Finished:</strong>
+          9 days ago
+        </p>
+        <Hds::Table @caption="Example table">
+          <:head as |H|>
+            <H.Tr>
+              <H.Th>Name</H.Th>
+              <H.Th>Type</H.Th>
+              <H.Th>Value</H.Th>
+            </H.Tr>
+          </:head>
+          <:body as |B|>
+            <B.Tr>
+              <B.Td>Cell one A</B.Td>
+              <B.Td>Cell two A</B.Td>
+              <B.Td>Cell three A</B.Td>
+            </B.Tr>
+            <B.Tr>
+              <B.Td>Cell one B</B.Td>
+              <B.Td>Cell two B</B.Td>
+              <B.Td>Cell three B</B.Td>
+            </B.Tr>
+          </:body>
+        </Hds::Table>
+      </:content>
+    </A.Item>
+
+    <A.Item @containsInteractive={{false}} @isOpen={{true}}>
+      <:toggle>
+        <div class="hds-typography-body-300">
+          Text inside a nested div with <a href="https://www.hashicorp.com/">a link</a>.
+        </div>
+      </:toggle>
+      <:content>
+        <p>
+          <Hds::Form::TextInput::Field @type="email" as |F|>
+            <F.Label>Email</F.Label>
+          </Hds::Form::TextInput::Field>
+        </p>
+        <Hds::Button @text="Submit" />
+      </:content>
+    </A.Item>
+  </Hds::Accordion>
+```
+
+### isOpen
+
+Set `isOpen` to `true` on an AccordionItem to display its associated content on page load instead of initially hiding it.
+
+```handlebars
+  <Hds::Accordion as |A|>
+    <A.Item @isOpen={{true}}>
+      <:toggle>Item one</:toggle>
+      <:content>
+        Additional content for item one which is displayed on page load
+      </:content>
+    </A.Item>
+    <A.Item>
+      <:toggle>Item two</:toggle>
+      <:content>
+        Additional content for item two which is hidden on page load
+      </:content>
+    </A.Item>
+  </Hds::Accordion>
+```
+
+### containsInteractive
+
+By default, the `containsInteractive` property of the AccordionItem is set to `false`, meaning that the entire AccordionItem toggle block can be clicked to hide and show the associated content. If set to `truee` only the chevron button of the AccordionItem is clickable vs. the entire block. This allows you to add other interactive content inside the toggle block if desired.
+
+```handlebars
```

Comment:
I agree with Alex that we should have more realistic examples here in the "How to use" (probably based on the audit examples). Not too complex, but they need to give an idea of what they may find in real use cases (eg. we've found some cases of buttons, and you already have an example in the showcase that you can re-use here).

Reply:
In general I agree with the approach you describe, is the right one. But in this specific case, there's no example that can likely match what they really need: in any case they have to copy, paste and modify it to adapt to their specific designs. Which means all the examples are equivalent. 
So if they all are equivalent, better to use examples that resemble something we have seen in the audit, like this one
<img width="508" alt="screenshot_2888" src="https://github.com/hashicorp/design-system/assets/686239/0b4171f3-27c4-4924-8086-33b894a8d2b8">
or this one
<img width="615" alt="screenshot_2887" src="https://github.com/hashicorp/design-system/assets/686239/41b37030-7997-43ab-b99f-e45861b03e9f">

below I'm adding a suggestion, so you can see what I mean